src/store/modules/getters/staticData.js

```diff
@@ -1,6 +1,9 @@
 import { isWritingNFT } from '~/util/nft';
 
-export const getUserInfoById = state => id => state.userInfos[id];
+export const getUserInfoById = state => id => state.userInfosById[id];
```

Comment:
Should remain unchanged for this PR?
```suggestion
export const getUserInfoById = state => id => state.userInfos[id];
```